airflow/jobs.py

```diff
@@ -1272,7 +1272,7 @@ def _log_file_processing_stats(self,
             if last_runtime is not None:
                 Stats.gauge('last_runtime.{}'.format(name), last_runtime)
             if last_run is not None:
-                unixtime = last_run.strftime("%s")
+                unixtime = int(last_run.strftime("%s"))
```

Comment:
what is this change?

Reply:
statsd only takes `int` and we are passing `str` here so it was constantly failing